pkg/analysis/passes/coderules/coderules.go

```diff
@@ -75,8 +75,6 @@ func run(pass *analysis.Pass) (interface{}, error) {
 	// run semgrep against the source code
 	semGrepArgs := []string{
 		"--json",
-		"--lang",
```

Comment:
This what the problem with the update. Since the semgrep rules file already contain which language files should handle, this parameter is redundant and it was considered a "bug" for semgrep to accept it,.